src/pal/src/exception/remote-unwind-xdac.cpp

```diff
@@ -82,10 +82,12 @@ typedef BOOL(*UnwindReadMemoryCallback)(PVOID address, PVOID buffer, SIZE_T size
 #define ERROR(x, ...)
 
 
-#ifdef TARGET_64BIT
+#ifdef BIT64
```

Comment:
Just out consistency, should `defined(BIT64)` be used here?

Reply:
Good catch.  I think it is better style.